### PR TITLE
Multicluster observables related to axis

### PR DESCRIPTION
**Pull request by artlbv**
This is to replace @beaudett's PR https://github.com/CMS-HGCAL/reco-ntuples/pull/33. I've taken his branch and:
- fixed few things (NB! including global indentation and whitespaces)
-  added `sigmaEtaEta/PhiPhi` for multiclusters
- implemented @clelange last suggestion about a flag to save these variables:`storePCAvariables` is `false` by default

I've tested the changes and the flag, and it seems to work fine.

### CODE REVIEW

File: HGCalAnalysis/plugins/HGCalAnalysis.cc [hunk 1]
```diff
@@ -448,622 +510,683 @@ HGCalAnalysis::~HGCalAnalysis()
...
+                }
+                xp.push_back(xp_curr);
+                yp.push_back(yp_curr);
+                zp.push_back(zp_curr);
+            } // closes loop on layers
+        } // closes conditions pt>3
+
+        // save info in tree
+        track_pt.push_back(it_track->pt());
+        track_pt.push_back(it_track->eta());
```
Comment (clelange) [inline HGCalAnalysis/plugins/HGCalAnalysis.cc]: You don't use the latest version of the code, this has already been fixed and now you're re-introducing the bug.

**Comment by clelange** [comment]
Also mind that this branch has conflicts, probably because of the code I pointed out above.

File: HGCalAnalysis/plugins/HGCalAnalysis.cc (hunk 1 above)
Comment (artlbv) [inline HGCalAnalysis/plugins/HGCalAnalysis.cc]: I don't know how this has happened, but once I've skipped my own indentation commit, it was fixed again

**Comment by artlbv** [comment]
I've recreated the commits skipping the indentation commit, please see PR #35

**Pull request closed without merging**